### PR TITLE
add pycaret

### PR DESCRIPTION
**Pull request by melonhead901**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pycaret/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  sha256: 5db9060f9d8ba53dbfb558b41de822c5b4175223c05a3a2c3c41c71346510694
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - catboost
```
Comment (synapticarbors) [inline recipes/pycaret/meta.yaml:19]: `host` should probably only include `python` and `pip`.

Comment (melonhead901) [reply recipes/pycaret/meta.yaml]: Done

File: recipes/pycaret/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5db9060f9d8ba53dbfb558b41de822c5b4175223c05a3a2c3c41c71346510694
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  # Limitations here because of xgboost package.
+  skip: true  # [win or linux32 or py2k]
```
Comment (scopatz) [inline recipes/pycaret/meta.yaml:16]: Can this be noarch python? We don't build on linux32, py2k can be solved with `python>=3` below and win probably doesn't have all the deps

Comment (melonhead901) [reply recipes/pycaret/meta.yaml:16]: I added `noarch: python`. I had that originally but the windows build failed so I removed it and added the skip line, which I got from the xgboost package (the dep that doesn't exist on windows)

Comment (melonhead901) [reply recipes/pycaret/meta.yaml:16]: actually nevermind - noarch won't work which selectors. Should I leave noarch in even though the windows version doesn't build?

Comment (scopatz) [reply recipes/pycaret/meta.yaml:16]: noarch packages only need to build on linux.  It is OK if windows fails.  Please make restore the noarch :wink:

Comment (melonhead901) [reply recipes/pycaret/meta.yaml]: Done!

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**